### PR TITLE
fix ir blaster

### PR DESCRIPTION
**Pull request by vo-1**
go

### CODE REVIEW

**Comment by MacTavishAO** [comment]
Tomorrow will build new release with your changes. If IR gets fixed will accept your req.

File: rootdir/init.qcom.rc [hunk 1]
```diff
@@ -92,6 +92,8 @@ on boot
     chown system system /sys/kernel/debug/msm_core/enable
     chown system system /sys/kernel/debug/msm_core/ptable
     chown system system /sys/kernel/boot_slpi/ssr
+    chown system system /sys/class/leds/infrared/transmit
```
Comment (Coderessurect) [inline rootdir/init.qcom.rc:95]: No need for this path as spidev 6.1 node can take care of it

File: rootdir/init.qcom.rc.save [hunk 2]
```diff
@@ -0,0 +1,573 @@
+# Copyright (c) 2009-2012, 2014-2017, The Linux Foundation. All rights reserved.
+#
```
Comment (Coderessurect) [inline rootdir/init.qcom.rc.save:2]: WTF why do we need this

File: rootdir/ueventd.qcom.rc [hunk 3]
```diff
@@ -280,5 +280,5 @@
 # IR
 /dev/lirc0                0660   system     system
 /dev/peel_ir              0660   system     system
-/dev/spidev7.1            0660   system     system
```
Comment (Coderessurect) [inline rootdir/ueventd.qcom.rc:283]: I rather prefer chmod this in init.rosy.rc as we have two nodes for spi interface control

**Comment by Coderessurect** [comment]
This pull request is not needed as these changes are not right

File: rootdir/init.qcom.rc.save (hunk 2 above)
Comment (vo-1) [reply rootdir/init.qcom.rc.save:2]: this is garbage file

**Pull request closed without merging**